.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,10 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
```

Comment:
ESLint parses all `.json` files as pure JSON (i.e. without comments), so a lint error is reported here. Does Dev Container also support this file with a `.jsonc` file extension? That would make the distinction more clear.

In either case, please add a section to `eslint.config.js`:

* If `.devcontainer/devcontainer.jsonc` is supported:
  
  ```js
  {
    files: [`**/*.jsonc`],
    rules: {
      'jsonc/no-comments': `off`,
    },
  },
  ```
* If only `.devcontainer/devcontainer.json` is supported:
  
  ```js
  {
    files: [`.devcontainer/devcontainer.json`],
    rules: {
      'jsonc/no-comments': `off`,
    },
  },
  ```

Reply:
@FloEdelmann Unfortunately, `.jsonc` is not supported by the specification, see https://containers.dev/implementors/spec/#devcontainerjson

> In either case, please add a section to eslint.config.js

You mean the `overrides` section in `.eslintrc.cjs` right?

Reply:
Ah, just saw that the file changed yesterday to `eslint.config.js`

Reply:
I actually migrated to ESLint v9 with `eslint.config.js` yesterday, see #4302 :sweat_smile: 

So your branch was likely out of date and still missing that change, that's why there is a conflict now. Could you please update it? (The actual change in [`82090c5`](https://github.com/OpenLightingProject/open-fixture-library/pull/4222/commits/82090c5e74d88e5a40e5b84e9d4d26dfc00e3aec) looks fine.)

Reply:
Yeah I force pushed just now :)